instrumentation-tests/src/androidTest/java/com/mapbox/navigation/instrumentation_tests/core/MapboxHistoryTest.kt

```diff
@@ -79,6 +78,8 @@ class MapboxHistoryTest : BaseTest<EmptyTestActivity>(EmptyTestActivity::class.j
 
         // execute
         runOnMainSync {
+            mapboxNavigation.historyRecorder.startRecording()
+
```

Comment:
Add a pushHistory custom event here. And then verify the read position in the verifyHistoryEvents function